app/src/main/java/fr/free/nrw/commons/nearby/NearbyPlaces.java

```diff
@@ -124,7 +124,7 @@ public NearbyPlaces() {
 
             places.add(new Place(
                     name,
-                    Place.Description.fromText(type), // list
+                    Place.Label.fromText(type), // list
```

Comment:
Fixes for the longDescription bug - this was already submitted at #1034 but needs to be re-done here due to how much the Nearby code has changed in this branch